extensions/amp-form/0.1/form-validators.js

```diff
@@ -74,6 +75,12 @@ export class FormValidator {
 
     /** @protected @const {!Document} */
     this.doc = /** @type {!Document} */ (form.ownerDocument);
+
+    /**
+     * Tribool indicating last known validity of form.
+     * @private {boolean|null}
+     */
+    this.formValidity_ = null;
```

Comment:
Alrighty, I suppose it's just up to personal preference of `null` having an implicit meaning vs an explicitly-named three-value state enum with an `INITIAL` state. You do you